example/integration_test/vendor_test.dart

```diff
@@ -25,8 +26,36 @@ void main() {
   final listKey = Key("components_list");
 
   // Native message strings.
-  final vendorNames = "Exponential Interactive, Inc d/b/a VDX.tv, Index Exchange, Inc. , Fifty";
-  final notReadyMessage = "Native message: Failed: 'Didomi SDK is not ready. Use the onReady callback to access this method.'.";
+  const emptyDisabledVendorMessage = "Native message: Disabled Vendor list is empty.";
+  const emptyEnabledVendorMessage = "Native message: Enabled Vendor list is empty.";
+  const suffixVendorMessage = "1, 10, 100, 1000, 1001, 1002, 1003, 1004, 1006, 1007, 101, 1010, 1011, 1012, 1015, 1016, 102, 104, 108, 109, 11, 110, "
```

Comment:
maybe these constants could go in the constants file too?

Reply:
As they are used only here it seems unappropriated.